bindings/python/build_tools/cmake/iree_pybind_cc_library.cmake

```diff
@@ -0,0 +1,123 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+include(CMakeParseArguments)
+
+# iree_pybind_cc_library()
+#
+# CMake function to imitate Bazel's pybind_cc_library rule.
+#
+# Parameters:
+# NAME: name of target
+# HDRS: List of public header files for the library
+# SRCS: List of source files for the library
+# DEPS: List of other libraries to be linked in to the binary targets
+# COPTS: List of private compile options
+# DEFINES: List of public defines
+# INCLUDES: Include directories to add to dependencies
+# LINKOPTS: List of link options
+# TYPE: Type of library to be crated: either "MODULE", "SHARED" or "STATIC" (default).
```

Comment:
I take it this relates to the discussion at https://groups.google.com/forum/#!topic/iree-discuss/k0vhsm_tB54 ? As there isn't a Bazel equivalent (yet?) for this parameter, this will be another point that the build systems will diverge a bit and a Bazel -> CMake script won't be able to run without a bit of supervision.

Reply:
Yes, this indeed related to the linked discussion. You're right, this introduces a diverge, but the discussion showed out that it's not intended to take over Bazels behavior.